version/version.go

```diff
@@ -8,33 +8,29 @@ import (
 
 // Variables set by build flags
 var (
-	Commit        = ""
-	Version       = ""
-	VendorDirHash = ""
-	BuildTags     = ""
+	Commit    = ""
+	Version   = ""
+	BuildTags = ""
 )
 
 type versionInfo struct {
-	CosmosSDK     string `json:"cosmos_sdk"`
-	GitCommit     string `json:"commit"`
-	VendorDirHash string `json:"vendor_hash"`
-	BuildTags     string `json:"build_tags"`
-	GoVersion     string `json:"go"`
+	CosmosSDK string `json:"cosmos_sdk"`
+	GitCommit string `json:"commit"`
+	BuildTags string `json:"build_tags"`
+	GoVersion string `json:"go"`
 }
 
 func (v versionInfo) String() string {
 	return fmt.Sprintf(`cosmos-sdk: %s
 git commit: %s
-vendor hash: %s
```

Comment:
Ditto. Renaming might be ok. Removal is most definitely _not_.

Reply:
Rename to `gomod hash`